.github/workflows/main.yml

```diff
@@ -52,8 +52,7 @@ jobs:
         toolchain:
           - linux-gcc
           - macos-clang
-          - windows-msvc-shared
-          - windows-msvc-static
+          - windows-msvc
```

Comment:
why don't we longer test msvc shared vs static?

Reply:
I find it would still be valuable to keep a static config, to ensure all works fine in this config